Dockerfile

```diff
@@ -17,3 +12,8 @@ ENTRYPOINT ["sh", "/opt/flask.sh" ]
 FROM base as test
 ENV PATH="${PATH}:/root/todo_app"
 CMD ["poetry", "run", "pytest"]
+
+FROM base as production
```

Comment:
It might be worth a comment to say your CD pipeline assumes that the production container is the last target in the Dockerfile